projects/schematics/src/modernize-app-migrated-from-6_8-to-2211_19/ssr/update-server-ts/remove-webpack-from-server-ts.ts

```diff
@@ -0,0 +1,128 @@
+/*
+ * SPDX-FileCopyrightText: 2025 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { findNodes } from '@schematics/angular/utility/ast-utils';
+import { parseTsFileContent } from '../../../shared/utils/file-utils';
+import { removeVariableDeclaration } from '../../../shared/utils/variable-utils';
+import * as ts from 'typescript';
```

Comment:
`nitpick:` this could be converted to a default import.

Reply:
done